pkg/reconciler/buildtemplate/resources/names/names_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package names
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/knative/build/pkg/apis/build/v1alpha1"
+)
+
+func TestNamer(t *testing.T) {
+	tests := []struct {
+		name string
+		bt   *v1alpha1.BuildTemplate
+		f    func(*v1alpha1.BuildTemplate) string
+		want string
+	}{{
+		name: "simple ImageCache test",
```

Comment:
I made a similar comment in a PR on knative/pkg, but I don't think having two runs at this test adds anything, and might confuse people who think the two cases are testing different things.

I'd prefer just one test that checks that tests/demonstrates how the format is applied.

Reply:
Done.